packages/frontend/src/components/UserSettings/UserManagementPanel/index.tsx

```diff
@@ -190,7 +190,7 @@ const UserListItem: FC<{
                                         <Title order={4}>Delete user</Title>
                                     </Group>
                                 }
-                                yOffset="30vh"
+                                centered
```

Comment:
is this part of the PR? its ok if its not an accidental change

Reply:
it's part of the PR, I was prev using yOffset cause I missed this prop so I fixed it there too for consistency